### PR TITLE
Fix docker-compose

### PR DESCRIPTION
**Pull request by mkontani**
docker-compose settings is wrong and not work now.

I fixed compose and healthcheck like below `healthy`.

```sh
ᐅ docker ps
CONTAINER ID   IMAGE         COMMAND                  CREATED              STATUS                        PORTS                    NAMES
410cdfe981a5   mobtime_app   "docker-entrypoint.s…"   About a minute ago   Up About a minute (healthy)   0.0.0.0:1234->1234/tcp   mobtime-app
55033d8c9a23   redis         "docker-entrypoint.s…"   27 minutes ago       Up About a minute             6379/tcp                 cache
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,7 +4,9 @@ WORKDIR /web
 
 ADD . ./
 
+RUN apk add curl
+
 RUN npm ci
 RUN npm run tailwind
 
-CMD [ "npm", "run", "dev"]
+CMD [ "npm", "start"]
```
Comment (mrozbarry) [inline Dockerfile:12]: This was intentionally `npm run dev`. If you want to use docker in production, I'd be open to having a `Dockerfile.local` and `Dockerfile.production`, though.

File: docker-compose.yml [hunk 2]
```diff
@@ -24,8 +23,6 @@ services:
     ports:
       - 0.0.0.0:1234:1234
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost"]
+      test: ["CMD", "curl", "-f", "http://localhost:1234"]
```
Comment (mrozbarry) [inline docker-compose.yml:26]: Oh, good catch!

File: docker-compose.yml [hunk 3]
```diff
@@ -24,8 +23,6 @@ services:
...
     depends_on:
       - redis
-    volumes:
```
Comment (mrozbarry) [inline docker-compose.yml:30]: With this removed, how does docker have any access to the code? I'm not a docker expert, so you're going to have to spell it out for me.

**Review by mrozbarry — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (mkontani) [reply Dockerfile:12]: Oh, I understood. You mean this Dockerfile is for develop on local?
I'll fix.

File: docker-compose.yml (hunk 3 above)
Comment (mkontani) [reply docker-compose.yml:30]: If this compose is for dev on local, this volumes is necessary for propergating local changes into docker,
but in production case,  codes are added in Dockerfile [ADD step](https://github.com/mobtimeapp/mobtime/pull/266/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557L5), so this volume is unnecessary.

File: Dockerfile (hunk 1 above)
Comment (mrozbarry) [reply Dockerfile:12]: Yeah, that's what I've been doing. My deploy server just has all the dependencies installed, so I've had no need for docker in production, but I will gladly support it if people are deploying their own copies.

File: docker-compose.yml (hunk 2 above)
Comment (mrozbarry) [reply docker-compose.yml:26]: I don't mind adding curl for this, but any idea if we can do this without adding the curl dependency? I'm not opposed if this is the most straight-forward, just thinking out loud.

Comment (mkontani) [reply docker-compose.yml:26]: The default docker image node-alpine doesn't contain cli tool for checking, so any tool like curl/wget is usually added with.
or If we don't use healthcheck result,  we may delete healthcheck setting and curl.

File: docker-compose.yml (hunk 3 above)
Comment (mrozbarry) [reply docker-compose.yml:30]: Okay, I think we should have a `docker-compose.yml` for base settings, and `docker-compose.development.yml` and `docker-compose.production.yml` to override for their environments. Is that alright? I think then we can use the volume for development and skip it for production.

File: docker-compose.yml (hunk 2 above)
Comment (mrozbarry) [reply docker-compose.yml:26]: Alright, I think keeping the healthcheck and curl are good 👍

File: docker-compose.yml (hunk 3 above)
Comment (mkontani) [reply docker-compose.yml:30]: Ok, `docker-compose.development.yml` uses volume, and `docker-compose.production.yml` does not.

What's the differences between `docker-compose.yml` and `docker-compose.development.yml`?

Comment (mkontani) [reply docker-compose.yml:30]: thanks desc, I understood!

File: Dockerfile.production [hunk 4]
```diff
@@ -0,0 +1,15 @@
+FROM node:lts-alpine
```
Comment (mkontani) [inline Dockerfile.production:1]: I think maybe it is useful if this docker image is published to any registry like hub.docker.com or ghcr.io, 
but this PR is out of scope.
